corehq/apps/reports/models.py

```diff
@@ -1036,12 +1039,7 @@ def case_exports(self):
 
     @classmethod
     def by_domain(cls, domain):
-        return cache_core.cached_view(cls.get_db(), "groupexport/by_domain",
-            key=domain,
-            reduce=False,
-            include_docs=True,
-            wrapper=cls.wrap,
-        )
+        return hq_group_export_configs_by_domain(domain, cls)
```

Comment:
Hey I wrote up a little FAQ on this just now, https://gist.github.com/dannyroberts/460abd70be3d271fbc88. There are a couple more things you have to do do to get cache invalidation to work